pkg/plugins/envvars/envvars_test.go

```diff
@@ -671,6 +671,19 @@ func TestInitalizer_azureEnvVars(t *testing.T) {
 	})
 }
 
+func TestService_GetConfigMap_Defaults(t *testing.T) {
+	s := &Service{
+		cfg: &config.Cfg{},
+	}
+
+	require.Equal(t, map[string]string{
+		"GF_SQL_ROW_LIMIT":                         "0",
+		"GF_SQL_MAX_OPEN_CONNS_DEFAULT":            "0",
+		"GF_SQL_MAX_IDLE_CONNS_DEFAULT":            "0",
+		"GF_SQL_MAX_CONN_LIFETIME_SECONDS_DEFAULT": "0",
+	}, s.GetConfigMap(context.Background(), "", nil))
+}
```

Comment:
this test will keep failing when adding new properties or if you want to ensure that these properties are set by default, use  `require.Subset` as you did in other tests

Reply:
you're right about this failing when more are added, i kind of assumed people will add to this every time they do so.
i added this test based on @wbrowne 's comment at https://github.com/grafana/grafana/pull/83071#pullrequestreview-1888723326 , so i'm up to changing this in any way. 

if we go `Subset`, i'd just go for a single-value to illustrate that the default-config is not empty.. perhaps even a test for `len(keys) > 0`? WDYT?

Reply:
I'm afraid that by using subset everywhere that we will miss testing the entirety of the config. As long as we are able to test this somehow, I don't really mind how it looks

Reply:
it's a tradeoff, a test that will test the entire config will constantly fail when adding new properties. As far as we cover the whole config with multiple test, focusing on specific sections I think we are fine but if we want to be extra-careful we can have a test for the whole config, checking that we are not adding unexpected things.

Reply:
>a test that will test the entire config will constantly fail when adding new properties.

IMO that's what we want though. I'd like people to realize when they're adding a config that will always be set or not. The specific sections are fine, but a single test is important. It makes sense in my head to have this test case first and then the rest tests the subsets.